brave/browser-laptop
# Replaced fa-times for close button on download bar with SVG button

Closes #6070

Auditors: @bbondy @bradleyrichter

Test Plan:
1. Download something
2. Make sure the close button on the download bar works

![close2](https://cloud.githubusercontent.com/assets/3362943/20978782/805dc196-bced-11e6-94f0-6e1cafd78363.gif)
